BUILD

```diff
@@ -2820,6 +2820,17 @@ grpc_cc_library(
     ],
 )
 
+grpc_cc_library(
```

Comment:
do we need this target?

Reply:
ultimately i'd like to get to a point where we're separated out far more than we are: it means we'll be able to write small unit tests that can run while the rest of the system builds, which will speed up our presubmit times... gotta start somewhere